ara/ui/templates/partials/task_status_icon.html

```diff
@@ -0,0 +1,6 @@
+<a
+  {% if not static_generation %}href="{% url 'ui:task_index' %}?status={{ status }}"{% endif %}
+  class="btn btn-sm {% if status == 'completed' %}btn-success{% elif status == 'failed' %}btn-danger{% elif status == 'running' %}btn-info{% else %}btn-warning{% endif %}"
```

Comment:
There's no failed status for a task:
```suggestion
  class="btn btn-sm {% if status == 'completed' %}btn-success{% elif status == 'running' %}btn-info{% else %}btn-warning{% endif %}"
```